thirdparty/runtime_environments.go

```diff
@@ -58,3 +64,33 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// getOSInfo returns a list of operating system changes (name and version) from the corresponding AMI id.
```

Comment:
nit. small revision to comment:
```suggestion
// getOSInfo returns a list of operating system information for an AMI.
```